From bb151b985a05531efcca3b85503c5a9443eb60d4 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:34:52 -0400
Subject: [fs] bio: don't write "bio: create slab" messages to syslog

Message-id: <1402670238-13916-5-git-send-email-snitzer@redhat.com>
Patchwork-id: 83761
O-Subject: [RHEL7.1 PATCH 004/150] bio: don't write "bio: create slab" messages to syslog
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit b4d7124b2f2e29541e5c8815bd84ea55158dd730
Author: Mikulas Patocka <mpatocka@redhat.com>
Date:   Tue Feb 18 13:09:34 2014 -0500

    bio: don't write "bio: create slab" messages to syslog

    When using device mapper, there are many "bio: create slab" messages in
    the log. Device mapper targets have different front_pad, so each time when
    we load a target that wasn't loaded before, we allocate a slab with the
    appropriate front_pad and there is associated "bio: create slab" message.

    This patch removes these messages, there is no need for them.

    Signed-off-by: Mikulas Patocka <mpatocka@redhat.com>
    Signed-off-by: Jens Axboe <axboe@fb.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/bio.c b/fs/bio.c
index 5e7507d..f272171 100644
--- a/fs/bio.c
+++ b/fs/bio.c
@@ -118,7 +118,6 @@ static struct kmem_cache *bio_find_or_create_slab(unsigned int extra_size)
  if (!slab)
   goto out_unlock;
 
- printk(KERN_INFO "bio: create slab <%s> at %d\n", bslab->name, entry);
  bslab->slab = slab;
  bslab->slab_ref = 1;
  bslab->slab_size = sz;
-- 
1.7.1